lib/vimeo/index.js

```diff
@@ -25,18 +25,34 @@ const getVimeoVideoById = async id => {
     throw new Error(`Vimeo API Error: ${response.error}.`)
   }
 
+  const mp4 =
+    response.files &&
+    response.files
+      .filter(file => file.type === 'video/mp4')
+      .sort((a, b) => a.width < b.width)[0].link_secure
```

Comment:
always use descending/ascending helper from [d3-array](https://github.com/d3/d3-array#descending)—more readable and handles equal values properly

Reply:
Done.